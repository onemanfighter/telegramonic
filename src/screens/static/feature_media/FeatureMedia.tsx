const FeatureMedia = () => {
  return (
    <div>
      <h1>Feature Media</h1>
    </div>
  );
};

export default FeatureMedia;
