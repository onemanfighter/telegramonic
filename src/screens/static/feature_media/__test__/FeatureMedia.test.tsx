import { render } from '@testing-library/react';
import FeatureMedia from '../FeatureMedia';

describe('FeatureMedia screen', () => {
  it('should render correctly', () => {
    const { container } = render(<FeatureMedia />);

    expect(container).toMatchSnapshot();
  });
});
