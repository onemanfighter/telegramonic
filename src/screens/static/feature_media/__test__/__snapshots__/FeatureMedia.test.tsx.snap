// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureMedia screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Feature Media
    </h1>
  </div>
</div>
`;
